src/components/views/right_panel/UserInfo.tsx

```diff
@@ -258,7 +270,7 @@ function DevicesSection({
             // cross-signing so that other users can then safely trust you.
             // For other people's devices, the more general verified check that
             // includes locally verified devices can be used.
-            const isVerified = isMe ? deviceTrust.isCrossSigningVerified() : deviceTrust.isVerified();
+            const isVerified = deviceTrust && (isMe ? deviceTrust.crossSigningVerified : deviceTrust.isVerified());
```

Comment:
```suggestion
            const isVerified = isMe ? deviceTrust?.crossSigningVerified : deviceTrust?.isVerified());
```

To match L177 in the same file

Reply:
The problem here is avoiding the implicit cast to boolean if `deviceTrust` is nullish. We could do something like `isVerified = !!(isMe ? deviceTrust?.crossSigningVerified : deviceTrust?.isVerified()))` but I'm far from convinced that's clearer.

At L177, we have an explicit check on `deviceTrust`, so I'd say my variant of this is closer to L177 than yours.